autofill/autofill-impl/src/main/res/drawable/ic_baseline_search_24.xml

```diff
@@ -15,7 +15,6 @@
   -->
 
 <vector android:height="24dp"
-    android:tint="#000000"
```

Comment:
icons should not be tinted, but using the proper fillColor instead (attr/daxColorPrimaryIcon replaces toolbarIconColor here). I'll add a lint rule to flag it.